spec/integration/cli_ssl_spec.rb

```diff
@@ -5,21 +5,27 @@
 
   include Pact::IntegrationTestSupport
 
-  before :all do
-    @pid = start_server 4343, '--ssl', false
-  end
-
   it "should respond with SSL" do
+    pid = start_server 4343, '--ssl', false
     wait_until_server_started 4343, true
+    kill_server pid
   end
 
   it "sets the X-Pact-Mock-Service-Location header with https" do
+    pid = start_server 4343, '--ssl', false
     wait_until_server_started 4343, true
     response = connect_via_ssl 4343
     expect(response.headers['X-Pact-Mock-Service-Location']).to eq 'https://0.0.0.0:4343'
+    kill_server pid
   end
 
-  after :all do
-    kill_server @pid
+  it "should accept a sslcert and sslkey" do
```

Comment:
The kill server needs to be in an after block, not at the end of the test, because if the test fails, the server won't get killed.